tasks/bzr.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: Set etckeeper bzr whoami
+  become: yes
```

Comment:
The playbook is already expected to be run with `become: True`. No need to repeat for all tasks.